src/main/webapp/js/book-page-loader.js

```
@@ -0,0 +1,62 @@
+
+
+// Get ?book=XYZ parameter value
+const urlParams = new URLSearchParams(window.location.search);
+const parameterBookname = urlParams.get('book');
+console.log(19);
+console.log(parameterBookname);
+// const parameterBookname = "4a785b44-d3d7-47f7-a936-6c49137cbd53";
+
+// URL must include ?book=XYZ parameter. If not, redirect to homepage.
+if (!parameterBookname) {
+  window.location.replace('/');
+}
+
+/** Sets the page title based on the URL parameter bookname. */
+function setPageTitle() {
+  document.getElementById('page-title').innerText = parameterBookname;
+  document.title = parameterBookname + ' - Book Page'; // eslint-disable-line prefer-template
+}
+
+/** Fetches book title and adds it to the page. */
+function fetchBookTitle() {
+  const url = `/title?book=${parameterBookname}`;
+  fetch(url)
+    .then(response => response.text())
+    .then((response) => {
+      const bookContainer = document.getElementById('book-title');
+      let aboutBook = response;
+      if (response === '') {
+        aboutBook = 'No information yet.';
```

I think the sentence here should be more specific. Maybe something like 'No title added.' or 'This book has no title yet.'. But then again, a book should have a title. So I don't know, maybe we should throw an error instead.